spector/frontend/src/components/common/navbar/NavBar.js

```diff
@@ -9,9 +9,11 @@ import { useNavigate } from 'react-router';
 const NavBar = () => {
 
   let navigate = useNavigate();
+  let userID = sessionStorage.getItem("userID")// Retrieve userID with this
+
 
   function handleClick() {
-    navigate('profile')
+    navigate('profile/<user id>/')
```

Comment:
My bad - when I put `<user id>` in the issue I meant that you need to replace this with a variable (the userID variable)